projects/plugins/jetpack/extensions/blocks/latex-formula/block.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "w0/latex-formula",
+	"version": "0.1.0",
+	"title": "Latex Formula",
+	"category": "formatting",
+	"icon": "editor-code",
+	"description": "Create and display LaTeX formulas with live preview and easy editing.",
+	"example": {
+		"attributes": {
+			"latex": "\\frac{a}{b}"
+		}
+	},
+	"attributes": {
+		"latex": {
+			"type": "string",
+			"default": ""
+		},
+		"editing": {
+			"type": "boolean",
+			"default": true
+		}
+	},
+	"supports": {
+		"html": false
+	},
+	"textdomain": "jetpack",
+	"editorScript": "file:./index.ts",
+	"editorStyle": "file:./index.css",
+	"style": "file:./style-index.css",
+	"viewScript": "file:./view.ts"
```

Comment:
You should be able to do away with those, enqueuing those files is managed by Jetpack.